.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: cylc/release-actions/install-cylc-components@v1
         with:
           cylc_flow: true
-          cylc_flow_opts: 'empy'
+          cylc_flow_opts: ''
```

Comment:
If I look for this `cylc_flow_opts` variable in cylc-rose code it only exists in the yaml file. Is it needed?

Reply:
Results of https://github.com/cylc/cylc-rose/actions/runs/12049448140 suggest that it is, in fact, necessary.

Reply:
The [default](https://github.com/cylc/release-actions/blob/1edb53c088f836f1ef49d00c608926bf9296b880/install-cylc-components/action.yml#L59-L62) is `'all'` (equivalent to `pip install cylc-flow[all]`) which is unnecessary for these tests, leaving it as `''` (as you have done) is the correct thing to do.